### DISCUSSION TITLE
Plot Sizes

### DISCUSSION THREAD

**Discussion by l008com** [Chia-Network/chia-blockchain#8375] [General]
I need some clarification about the different plot sizes. 
I've been told on reddit and other places that there really is no benefit (ignoring that smaller sizes may get blocked someday) to making larger plots, other than you can mix large and small plots to "tetris" them in to the available drive space, better filling up the drive. 

And the perfect example is a simple 1 TB drive. Fill it with three K32's and three K33's, and you get 999.9 GB used and 172 MB free.

But doing the same thing with just K32's would give you ~979 GB in use and a whole ~21GB unused on the drive. 

So that's all good in well, makes sense, and its totally understandable. 

But recently I was told that there isn't actually any MORE solutions in the extra space used by larger plots. In other words, a K33 has exactly twice as many solutions in it as a K32, even though a K33 uses up approximately 8 more GB. 

First off, can I get some confirmation here? Is this true? 
If it IS true, then why wouldn't you ALWAYS use K32's? Those 108 GB blocks first off would always give you the max usage of your drive, no matter how much free space was left. AND on larger drives, you'd be getting MORE solutions squished in using all K32's then by doing any mixing and matching, as larger plot sizes have wasted space. 

So you'd get the most solutions filling an 18 GB drive with just K32's, knowing that in a few years, when K32's get dumped, you'd have to re-plot the drive. But if that's years away, it seems 100% worthwhile. 

So these two ideas above about how plotting works clearly are not both right. One has to be right and one has to be wrong (ok technically both could be wrong). But which is it?

**Comment by loppefaaret** [comment]
> First off, can I get some confirmation here? Is this true?

no
if we say a k33 is exactly 2.06 times larger than a k32, it holds 2.06 times the proofs

**Reply by l008com** [reply]
Is that what you say? k33 is 2.06X a k32? That math checks out based on disk sizes.

**Reply by loppefaaret** [reply]
it is what is generally said on our keybase, the size of a plot can vary a bit, depending on how the different tables chain together during the creation, but generally, that is the number

**Comment by BrandtH22** [comment]
@l008com , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
